src/VisualStudio/NuGet.Packaging.VisualStudio.15/Templates/Projects/Nuget/Package/Package.nuproj

```diff
@@ -14,6 +14,7 @@
 	</ItemGroup>
 	<PropertyGroup Label="Configuration">
 		<PackageId>$safeprojectname$</PackageId>
+		<PackageVersion>1.0.0</PackageVersion>
```

Comment:
Ah, that's true. Config and Platform were something of special cases. But I see that PackageVersion would also potentially be handy to set via env var. Good call.